### PR TITLE
improve e2e output and error log

### PR DESCRIPTION
**Pull request by calvix**
adding few extra log outputs to improve observability on which stage of e2e actually failed

also teardown stage could silently fail as ignored errors

### CODE REVIEW

File: integration/setup/setup.go [hunk 1]
```diff
@@ -38,14 +38,27 @@ func WrapTestMain(g *framework.Guest, h *framework.Host, m *testing.M) {
 
 	err := Setup(g, h)
 	if err != nil {
-		log.Printf("%#v\n", err)
+		log.Printf("\nSetup stage failed,")
```
Comment (MarcelMue) [inline integration/setup/setup.go]: These newlines look a bit all over the place to me, what does this output look like?

Comment (calvix) [reply integration/setup/setup.go]: extra new lines for some separation its mostly intended that each stage has atleast  1 empty line between each other and also including the separator so it should look like this:
```
<... setup stage output ... >
<empty line>
Setup stage finished/failed
<empty line>
< .... test stage output ... >
<empty line>
Test stage finished/failed
<empty line>
< ... teardown stage output ... >
<empty line>
Teardown stage finished/failed
```

Comment (calvix) [reply integration/setup/setup.go]: It makes sense to me, do you think its bad idea?

Comment (xh3b4sd) [reply integration/setup/setup.go]: Having the missing error handling makes sense. I am not sure about the logging though. This should actually be aligned with other projects and we should also use our logger.

Comment (xh3b4sd) [reply integration/setup/setup.go]: Ok, but we should use our logger and not another and another and do something different here and there and all ends up in a mess. So I would say lets please use our logger and do this in other projects as well.

**Comment by calvix** [comment]
removed empty lines to have less observability and used `micrologger` instead of `log`

File: integration/setup/setup.go [hunk 2]
```diff
@@ -35,17 +34,37 @@ const (
 // WrapTestMain setup and teardown e2e testing environment.
 func WrapTestMain(g *framework.Guest, h *framework.Host, m *testing.M) {
 	var r int
+	var l micrologger.Logger
```
Comment (xh3b4sd) [inline integration/setup/setup.go]: There is a logger already created. Check where the frameworks above come from. Lets just dispatch the logger from there to here.

Comment (calvix) [reply integration/setup/setup.go]: Where? I check all three structs and none of them has the micrologger property? Is it hidden somewhere?

I checked
```
g *framework.Guest
h *framework.Host
m *testing.M
```

Comment (xh3b4sd) [reply integration/setup/setup.go]: `WrapTestMain` is called with 3 arguments. Make 4 out of that.

**Pull request merged**